src/components/DashboardStation/DashboardStation.tsx

```diff
@@ -34,7 +34,7 @@ export function DashboardStation(props: IDashboardStationProps) {
     }
 
     loadImages();
-  }, [storedTheme]);
+  }, [storedTheme, components, imagesDark, imagesLight]);
```

Comment:
do you need all these ?

Reply:
components yes
for both image lists if someone wants to modify the images in the future. No preference